sci-mathematics/cgal/cgal-4.11.1.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+CMAKE_BUILD_TYPE=Release
+inherit cmake-utils flag-o-matic
+
+MY_P=CGAL-${PV}
+
+DESCRIPTION="C++ library for geometric algorithms and data structures"
+HOMEPAGE="http://www.cgal.org/"
+SRC_URI="
+	https://github.com/CGAL/cgal/releases/download/releases%2F${MY_P}/${MY_P}.tar.xz
+	doc? ( https://github.com/CGAL/cgal/releases/download/releases%2F${MY_P}/${MY_P}-doc_html.tar.xz )"
+
+LICENSE="LGPL-3 GPL-3 Boost-1.0"
+SLOT="0/13"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc examples +gmp mpfi ntl qt5"
+
+RDEPEND="
+	>=dev-cpp/eigen-3.1
+	dev-libs/boost:=[threads]
+	dev-libs/mpfr:0
+	sys-libs/zlib:=
+	x11-libs/libX11:=
+	virtual/glu:=
+	virtual/opengl:=
+	gmp? ( dev-libs/gmp:=[cxx] )
+	qt5? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		dev-qt/qtopengl:5
+		dev-qt/qtsvg:5
+		dev-qt/qtwidgets:5
+	)
+	mpfi? ( sci-libs/mpfi )
+	ntl? ( dev-libs/ntl )"
+DEPEND="${RDEPEND}
+	app-arch/xz-utils
+	virtual/pkgconfig"
+
+S="${WORKDIR}/${MY_P}"
+
+src_prepare() {
+	cmake-utils_src_prepare
+	# modules provided by dev-util/cmake and dev-cpp/eigen
+	rm cmake/modules/Find{Eigen3,GLEW,OpenGL}.cmake || die
+	sed -e '/install(FILES AUTHORS/d' \
+		-i CMakeLists.txt || die
+
+	# use C++11 threads instead of boost::thread
+	append-cxxflags -std=c++11
```

Comment:
Consequently, since this is actually 'downgrading' C++ detection in this package at this point, let's remove this line and see if anyone is going to complain. Alternatively raise to `c++14` (it is safe for all currently supported GCCs), your call.